ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -145,7 +147,7 @@ resources:
   type: git
   source:
     branch: ((geode-build-branch))
-    depth: 1
+    depth: 100
```

Comment:
Has there been an issue with the geode repo being too shallow for some task runs?

Reply:
Adding performance will require us to compare the performance of Geode at two different commits, so we need to check out more than just the most recent one or the commit needed for the "baseline" run will not be cloned.

Reply:
in that case, remove the `depth` flag altogether. There is no guarantee that 100 commits will be enough to get to the baseline SHA.